### ISSUE TITLE
Find which builds exist for branches/tags

### ISSUE THREAD

**Issue by bhdn** [aplanas/openSUSE-git#73]
Longer initial description: Be able to determine from a package git repository which builds (project git or non git) exist of these source branches/tags.

I see this more or less as a requirement for #46. @celan please confirm if this is the case.

**Comment by celan** [comment]
Yes, I can confirm this. That it is a requirement for #46 looks reasonable, too. Good catch!

**Comment by AdamMajer** [comment]
I believe this is the other way around from #46 and I'm not sure how feasible it is to do without additional functionality. You can map sources to any number of projects in OBS. To map these back, we would need some sort of API that can say

```
GET /org/package/SHA

OBS:project:1/package2
OBS:project:2/package2
OBS:project:3/package2
OBS:project:4/package2
```

**Comment by bhdn** [comment]
Just to record it here: Eduardo Navarro pointed in a private chat recently that on a project level we can match: `https://api.opensuse.org/search/project/id?match=scmsync%3D%27sometext%27)` and the same works (as I tested at least) for `/search/package/id`.
